lib/organization.js

```diff
@@ -1,18 +1,17 @@
 "use strict";
 const dopts = require("default-options");
 
-import { getValueFromLogs, requireContract } from "./utils.js";
-const Avatar = requireContract("Avatar");
-const Controller = requireContract("Controller");
+import { getDeployedContracts } from './contracts.js';
+import { getValueFromLogs, requireContract } from './utils.js';
+const Avatar = requireContract('Avatar');
+const Controller = requireContract('Controller');
 const DAOToken = requireContract("DAOToken");
 const GenesisScheme = requireContract("GenesisScheme");
 const Reputation = requireContract("Reputation");
 // import { GlobalConstraintRegistrar } from   './globalconstraintregistrar.js';
 const AbsoluteVote = requireContract("AbsoluteVote");
 // import { UpgradeScheme } from   './upgradescheme.js';
-
-import { getSettings } from "./settings.js";
-import { SchemeRegistrar } from "./schemeregistrar.js";
+import { SchemeRegistrar } from './schemeregistrar.js';
```

Comment:
I just checked in a big code reformatting PR that converted all the single quotes to double quotes, according to rules of prettify.  But I neglected to incorporate the rule into eslint.  I'm afraid we're going to have a lot of thrashing and more painful diff's if we don't settle on a standard with respect to quotes.

Reply:
not a big fan of prettify so far, but ok